requirements/base.txt

```diff
@@ -70,7 +70,7 @@ edx-rest-api-client==5.2.1  # via -r requirements/base.in, edx-ecommerce-worker
 enum34==1.1.10            # via cybersource-rest-client-python
 extras==1.0.0             # via cybersource-rest-client-python, python-subunit, testtools
 factory-boy==2.12.0       # via django-oscar
-faker==5.6.5              # via factory-boy
+faker==8.1.3              # via factory-boy
```

Comment:
This fixes:
https://github.com/eduNEXT/edunext-ecommerce/runs/3744167423?check_suite_focus=true